config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -109,7 +156,8 @@ presubmits:
         - test-unit
 
 periodics:
-- interval: 12h
+- interval: 8h
+  # ci-cloud-provider-azure-master runs Azure specific tests periodically.
```

Comment:
This is incorrect. This job has been updated to run ccm e2e tests. https://github.com/kubernetes/test-infra/blob/878b247b86e06822d5633a0a0f03d841b1f6b334/config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml#L194

Reply:
Right, those periodic jobs are ensuring there're no flaky things happen (if they are, we should fix them)